frontend/next.config.js

```diff
@@ -25,7 +25,7 @@ const moduleExports = {
     DISCORD_CLIENT_ID: process.env.DISCORD_CLIENT_ID,
     DISCORD_CLIENT_SECRET: process.env.DISCORD_CLIENT_SECRET,
     services: {
-      apiUrl: 'http://api.podkrepi.localhost:5000',
```

Comment:
@imilchev That's actually not a bug of hardcoded url but alias to the api container in the local docker-compose network


https://github.com/podkrepi-bg/frontend/blob/f58120463810906c72c9b009d04bfc842f2f4f2b/docker-compose.yml#L47-L49

If we'll use another `API_URL` we might define which that is - at the moment it seems to be `CONTACT_API_URL` pointing to the contact module

If we merge this version to `master` and deploy to current prod stack the support requests won't be available as the url will not be set in the deployment  https://github.com/podkrepi-bg/frontend/blob/f58120463810906c72c9b009d04bfc842f2f4f2b/.github/workflows/deploy-prod.yml#L125

Reply:
It's still strange that there is some hard-coded URL in this file. What is it supposed to point to? This will not work for sure on the cluster as we don't have a route to this URL so I guess we need another setting for it.

Reply:
@imilchev As I revised it there is an environment variable set in `.env.example` which refers to the `API_URL` with the correct value

https://github.com/podkrepi-bg/frontend/blob/bc8e1bb1867e1cb5d3ec172861fd485687bb1271/.env.example#L25-L26


The problem comes from the production env values of the same variable

https://github.com/podkrepi-bg/frontend/blob/bc8e1bb1867e1cb5d3ec172861fd485687bb1271/.env.prod#L24-L25

It points to `api.podkrepi.bg` which is invalid address at this point. If we want to use the `process.env.API_URL` we need to update it inside `.env.prod` to match the current hardcoded value so we don't break prod deployments.